### ISSUE TITLE
Token introspection - invalid token (9.11.2.02) (FI-3257)

### ISSUE THREAD

**Issue by cheng13231** [onc-healthit/onc-certification-g10-test-kit#572]
Test **9.11.2.02** fails for me when running g10 test suite locally, please see the screenshot below. 
![image](https://github.com/user-attachments/assets/cf7ae0b7-a2bb-4a9f-bf75-4f76d6657f97)

Given an invalid token, the test expects **200** response code, however **401** returns. I tried to **POST** a similar request to my authorization server, I do get **200** response, see screenshot below for the postman request
![image](https://github.com/user-attachments/assets/cd59f7a6-cfb8-44af-9e5a-ab9022facb35)

When I check the log in my authorization server, I find some differences, see screenshot below
![image](https://github.com/user-attachments/assets/b9a28fda-725a-4fb6-bea3-7afd56d8d4b2)

Seems that Test **9.11.2.02** doesn't properly send the `authorization` header provided in the test config to my authorization server. But, the strange thing to me is that Test **9.11.2.01** is passed. Could you please help me to check what I did wrong?

**Comment by yunwwang** [comment]
Hello @cheng13231

Thank you for reaching out to Inferno team. Does your server require Authorization header for token introspection endpoint?

**Comment by cheng13231** [comment]
Yes, it does, and I configured it in the setting "HTTP Authorization Header for Introspection Request". I would assume that the `authorization` header is used in test **9.11.2.01** as well, right?

**Comment by yunwwang** [comment]
We may have a bug in this test. To confirm that, one more question. Before you run the test, did you provide the authorization header for token introspection in this input field?

![image](https://github.com/user-attachments/assets/12ddffc3-11d9-4b17-8399-6cf0044c9521)

**Comment by AlthafGit** [comment]
Yeah, I am also getting the same issue. Inferno is not sending the Authorization in the header.

**Comment by AlthafGit** [comment]
@yunwwang  Still I am getting the same issue, How to solve this?

**Comment by VighneshAlevoorP** [comment]
@yunwwang: Is this issue resolved? Please let me know. I am still facing the issue.

**Comment by yunwwang** [comment]
Hello @VighneshAlevoorP @deepaktiwari29 :

This has been fixed in this PR: [FI-3257: Use Custom Authorization Header Input in Invalid Token Test inferno-framework/smart-app-launch-test-kit#81](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/81) and released in Inferno's November Release. 

Are you experiencing the same issue?
